src/ImageSharp/Formats/Jpg/Components/MutableSpan.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Buffers;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+
+namespace ImageSharp.Formats
+{
+    /// <summary>
+    /// Like corefxlab Span, but with an AddOffset() method for efficiency.
+    /// TODO: When Span will be official, consider replacing this class!
```

Comment:
I'm looking forward to being able to use it. I think it's get me out of some tricky situations I have with pixel rows in my Png codec where I want the byte array for unmanaged use but not the first item, plus I don't want to copy it.